### DISCUSSION TITLE
Laravel 9 Attribute [feeds] does not exits

### DISCUSSION THREAD

**Discussion by cihankusmez** [spatie/laravel-feed#187] [Bugs]
Hi,

Route:feeds() on web.php is working on localhost. But in production I'm recieving an error Attribute [feeds] does not exits. I'm using laravel 9 and I execute commands below after registering routes.

php artisan route:clear
php artisan config:clear
php artisan cache:clear

nothing changed. I had to comment Route::feeds() line 

What should I do ?

**Comment by enindu** [comment]
I Added `Spatie\Feed\FeedServiceProvider::class` into `providers` array in `config/app.php` file and fixed the issue
